src/index.html

```diff
@@ -15,8 +15,84 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
```

Comment:
Your attributes formatting is incorrect according to the checklist. When you have more than 2 attributes or long attribute values, start each one, including the first, on a new line with 2-space indentation related to the tag. Your tag’s closing bracket should be on the same level as the opening one.